lib/routes/hfut/hf/notice.ts

```diff
@@ -0,0 +1,42 @@
+import { Route } from '@/types';
+import parseList from './utils';
+
+export const route: Route = {
+    path: '/hf/notice/:type?',
+    categories: ['university'],
+    example: '/hfut/hf/notice/tzgg',
+    parameters: { type: '分类，见下表（默认为 `tzgg`)' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: true,
```

Comment:
Maybe I've missed something here, but I didn't see any signs of anti-crawling behaviour.